lib/dataContract/enrichDataContractWithBaseSchema.js

```diff
@@ -1,14 +1,19 @@
-const baseDocumentSchema = require('../../schema/base/document');
-
 const DataContract = require('./DataContract');
 
 /**
- * @typedef {enrichDataContractWithBaseDocument}
+ * @typedef {enrichDataContractWithBaseSchema}
+ *
  * @param {DataContract|RawDataContract} dataContract
+ * @param {Object} baseSchema
  * @param {string[]} [excludeBaseDocumentProperties]
+ *
  * @return {RawDataContract}
  */
-function enrichDataContractWithBaseDocument(dataContract, excludeBaseDocumentProperties = []) {
+function enrichDataContractWithBaseSchema(
+  dataContract,
+  baseSchema,
+  excludeBaseDocumentProperties = [],
```

Comment:
Do we use this option somewhere?

Reply:
@shumkov no, but, I would keep it for now